react-app/src/components/AppSideBar/AppSideBar.tsx

```diff
@@ -220,7 +220,9 @@ const AppSideBar: React.FC<AppSideBarProps> = (props) => {
             )}
           </div>
 
-          <main className={cn("flex", "flex-1", "min-w-0")}>{children}</main>
+          <main className={cn("flex", "flex-1", "min-w-0", "flex-col")}>
+            {children}
+          </main>
```

Comment:
Actually I think we want to change this back to `block` to enforce explicit styling for the children, do you think you can help with that? 🙏